packages/fluentui/react-northstar/src/components/Datepicker/Datepicker.tsx

```diff
@@ -107,6 +113,13 @@ export type DatepickerStylesProps = Pick<DatepickerProps, 'allowManualInput'>;
 
 export const datepickerClassName = 'ui-datepicker';
 
+export type accProps = {
```

Comment:
nit: better name ? `accProps` seems a bit mystic for those who might not be familiar with the codebase

Reply:
a11yProps/ariaProps would be more relevant IMO